modules/minimint-wallet/src/lib.rs

```diff
@@ -841,7 +841,7 @@ impl<'a> StatelessWallet<'a> {
         let peg_out_amount = outputs
             .iter()
             .map(|peg_out| peg_out.amount)
-            .fold1(|a, b| a + b)
+            .reduce(|a, b| a + b)
```

Comment:
Good catch!

Reply:
it was just `cargo check` complaining :smile:

Reply:
Ah, it probably didn't warn me or on CI because before the dependency update it wasn't deprecated.